convert/jenkinsjson/convertTestFiles/nunit/Jenkinsfile

```diff
@@ -0,0 +1,39 @@
+pipeline {
+    agent any
+    stages {
+        stage('Run NUnit') {
+            steps {
+            	writeFile(file: "TestResult.xml", text: '''
```

Comment:
I uploaded the xml content [here](https://github.com/harness-community/test-jenkins-to-harness/raw/refs/heads/main/nunit-test-result.xml) so instead of writing it to a file, you can add a step that downloads it and puts it in testResult.xml file